api/src/modules/User/model.ts

```diff
@@ -30,13 +27,13 @@ export const AuthPayload = objectType({
   name: 'AuthPayload',
   definition(t) {
     t.string('token');
-    t.field('user', { type: 'User' });
+    t.field('session', { type: 'Session' });
   },
 });
 
 export function getUserCrud(prisma: PrismaClient) {
   return {
-    signIn: async (userNamePass: { username: string; password: string }) => {
+    validatePassword: async (userNamePass: { username: string; password: string }) => {
```

Comment:
Nit: let's rename it to `getUserByUsernamePassword`, cause `validate` sounds like it doesn't return anything